src/Relations.dfy

```diff
@@ -1,12 +1,6 @@
-
-include "Helpers.dfy"
-
 module Relations { 
     export 
-    reveals Reflexive, AntiSymmetric, Connected, TotalOrdering
-    provides Helpers
-
-    import opened Helpers
+    reveals Reflexive, AntiSymmetric, Connected, TotalOrdering, Transitive, Trichotomous
 
     predicate Reflexive<T(!new)>(R: (T, T) -> bool) 
```

Comment:
I prefer that we unify our predicates and name them `IsReflexive`, `IsTransitive`, etc.